### PR TITLE
Add Austin to gigabit pages

### PR DESCRIPTION
**Pull request by alanmoo**
## Description
Update gigabit pages to include Austin, update links and images. As per https://github.com/mozilla/learning.mozilla.org/issues/1989

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1291014

## Testing
Content only changes. gulp js:test passes.

## Checklist
- [ ] Requires l10n changes.
- [ ] Related functional & integration tests passing.

Closes https://github.com/mozilla/learning.mozilla.org/issues/1989,
https://bugzilla.mozilla.org/show_bug.cgi?id=1291014

### CODE REVIEW

File: bedrock/gigabit/templates/gigabit/gigabit.html [hunk 1]
```diff
@@ -102,8 +102,9 @@ <h1 class="large">{{ _('Mozilla Gigabit Community Fund') }}</h1>
               {% endtrans %}
             </p>
             <div class="hive-logos">
-              <a href="http://hivekc.org/"><img src="{{ static('img/gigabit/Hive-Kansas-City.png') }}" alt="Hive Kansas City"></a>
-              <a href="http://hivecha.org/"><img src="{{ static('img/gigabit/Hive-Chattanooga.png') }}" alt="Hive Chattanooga"></a>
+              <a href="http://twitter.com/hiveatx"><img src="{{ static('img/gigabit/Hive-Austin.png') }}" alt="Hive Austin"></a>
```
Comment (alexgibson) [inline bedrock/gigabit/templates/gigabit/gigabit.html]: Nit: this could be `https://` also.

Comment (alanmoo) [inline bedrock/gigabit/templates/gigabit/gigabit.html]: Great catch, fix incoming.

File: bedrock/gigabit/templates/gigabit/gigabit.html [hunk 2]
```diff
@@ -115,7 +116,7 @@ <h1 class="large">{{ _('Mozilla Gigabit Community Fund') }}</h1>
             </p>
             <p>
               {% trans %}
-              In partnership with the National Science Foundation and US Ignite, we&rsquo;re expanding the Gigabit Community Fund to three additional gigabit cities by 2018. Interested in adding your community to list of communities being considered as part of this expansion? <a href="https://docs.google.com/forms/d/1DgjPFsydrQR2H_CdNrUEqOmbrWHASFZelV7WqQsGkRg/viewform">Please fill out this interest form</a>.
+              In partnership with the National Science Foundation and US Ignite, we&rsquo;re expanding the Gigabit Community Fund to two additional gigabit cities by 2018. Interested in adding your community to list of communities being considered as part of this expansion? <a href="https://docs.google.com/forms/d/1DgjPFsydrQR2H_CdNrUEqOmbrWHASFZelV7WqQsGkRg/viewform">Please fill out this interest form</a>.
```
Comment (alexgibson) [inline bedrock/gigabit/templates/gigabit/gigabit.html]: This wasn't introduced in this change, but ideally the `href` here would not be hard-coded in the trans block, and should use a variable instead like elsewhere in this template.

File: bedrock/gigabit/templates/gigabit/gigabit.html [hunk 3]
```diff
@@ -49,13 +49,13 @@ <h1 class="large">{{ _('Mozilla Gigabit Community Fund') }}</h1>
...
-              Since 2011, Mozilla has partnered with the <a href='{{ url_nsf }}'>National Science Foundation</a> and <a href='{{ url_us_ignite }}'>US Ignite</a> to demonstrate the value of gigabit networks for learning and to help seed demand for further high-speed network investment. In early 2014, the <a href='{{ url_hive_cha }}'>Hive Chattanooga</a> and <a href='{{ url_hive_kc }}'>Hive Kansas City</a> Learning Communities were launched to activate these leading gigabit cities as living laboratories for gigabit innovation. The Mozilla Gigabit Community Fund provides catalytic funding in these Hive communities to support the piloting of gigabit technologies and associated curricula. Intentionally designed to erase boundaries between educators and technologists, between organizations, and between gigabit cities, Gigabit Community Fund projects show the impact of next-generation networks on learning and engage new leaders as co-creators and beta-testers of gigabit-enabled educational technologies.
+            {% trans url_nsf='http://www.nsf.gov/', url_us_ignite='https://www.us-ignite.org/' %}
```
Comment (alexgibson) [inline bedrock/gigabit/templates/gigabit/gigabit.html]: Looks like this could be `https` also: https://www.nsf.gov/

**Comment by alexgibson** [comment]
Looks like you could resize `Hive-Austin.png` to 300px wide (currently over 1000px)

**Comment by alexgibson** [comment]
A few small nits, but otherwise looks good r+wc

**Comment by flukeout** [comment]
Here's a 300x142 px version of the logo we prepared for the Austin Hive...
https://dl.dropboxusercontent.com/u/109902/hive-austin-logo-300-142.png

**Comment by alanmoo** [comment]
@alexgibson I made those changes and added one line of css to align the logos.

**Comment by alexgibson** [comment]
Thanks again @alanmoo, r+

**Pull request merged**